packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -1727,7 +1733,17 @@ class CheckboxSetting extends StatelessWidget {
       children: [
         NotifierCheckbox(
           notifier: notifier,
-          onChanged: onChanged,
+          onChanged: (bool? value) {
+            final gaScreenName = this.gaScreenName;
+            final gaItem = this.gaItem;
+            if (gaScreenName != null && gaItem != null) {
+              ga.select(gaScreenName, gaItem);
+            }
+            final onChanged = this.onChanged;
+            if (onChanged != null) {
+              onChanged(value);
+            }
```

Comment:
I think conditional property access will accomplish the same effect here.
```suggestion
            this?.onChanged(value);
```